README.md

```diff
@@ -101,36 +100,56 @@ Prerequisites:
     export OASIS_GO=go1.13.8
     ```
 
-* [Rust](https://www.rust-lang.org) and the nightly toolchain.
+* [Rust].
 
-  Once you have [`rustup` installed](https://www.rust-lang.org/tools/install),
-  install the nightly with:
+  We follow [Rust upstream's recommendation][rust-upstream-rustup] on using
+  [rustup] to install and manage Rust versions.
 
-  ```
-  rustup install nightly
-  ```
+  _NOTE: [rustup] cannot be installed alongside a distribution packaged [Rust]
+  version. You will need to remove it before you can start using [rustup]._
```

Comment:
Doesn't it get weird to link every single use of a term?

Reply:
wikipedia says yes

Reply:
Yes, it is weird. I've fixed it.